File: exercicios/1541/1541.py
while True:
    n = input().split()
    if n == 0:
        break
    if n != 0:
        a = int(n[0])
        b = int(n[1])
        c = int(n[2])
    
    area = a * b
    total = (area * (c * 100))
    print(total)
    

File: exercicios/JavaScript/1541/1541.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n')

while(x!=0){

    var x = lines.shift().split(' ');

    var a = parseInt(x[0]);
    var b = parseInt(x[1]);
    var c = parseInt(x[2]);

    if(x == 0){
        break;
    }

    //console.log(`A = ${a}, b = ${b}, c = ${c}`);
    var A = a * b;
    var t = A * 100 / c;
    t = (t) ** (1/2);

    console.log(parseInt(t));
}